### ISSUE TITLE
🚀 Feature: Support for multiple GitHub webhook secrets

### ISSUE THREAD

**Issue by jawilson** [backstage/backstage#27598]
### 🔖 Feature description

Support to configure the [`events-backend-module-github`](https://github.com/backstage/backstage/tree/master/plugins/events-backend-module-github) plugin with multiple webhook secrets would be very useful as currently only one webhook secret can be used to validate incoming webhooks.

### 🎤 Context

I have a Backstage instance configured with two private GitHub apps, each installed in a different GitHub organization. Backstage handles this quite well already, but it does not support validating the webhook callbacks from both app installations if their webhook secret differs.

### ✌️ Possible Implementation

AFAICT the [`integrations.github.[].apps.[].webhookSecret`](https://github.com/backstage/backstage/blob/7bedf44bf6541a0b7eaeffb481a0ed4912a6bc55/packages/integration/src/github/config.ts#L95) is entirely unused. I propose using any configured GitHub app webhook secret values in addition to `events.modules.github.webhookSecret`, making the latter optional if any apps are configured.

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by vinzscam** [comment]
yeah it makes sense. Would you like to be assigned @jawilson?

**Comment by jawilson** [comment]
@vinzscam yup, I can take it. I already have the change written, just need to figure out a good way to test and go through the proper contribution guidelines.

**Comment by camilaibs** [comment]
Hello @jawilson  👋🏻, just a gentle ping.
Is it still possible for you to contribute with us?
It would be greatly appreciated if you could help us in resolving this issue 🙂 .

**Comment by pjungermann** [comment]
@jawilson are you still on that issue?

Your proposed solution sounds great to move forward with. If you are not on it anymore, I can give it a go.

**Comment by jawilson** [comment]
Sorry about the delay @camilaibs and @pjungermann. I still haven't had the chance to meet all of the contribution guidelines. That said, my changes are available [here](https://github.com/backstage/backstage/compare/master...blinemedical:backstage-fork:use-websocket-secret-from-github-app?expand=1) if someone would like to see it through.
